docs/azmcp-commands.md

```diff
@@ -681,12 +681,6 @@ azmcp group list --subscription <subscription>
 ### Azure Service Bus Operations
 
 ```bash
-# Peeks at messages in a Service Bus queue
-azmcp servicebus queue peek --subscription <subscription> \
```

Comment:
is this intentional ?

Reply:
Yes, these commands are not currently supported by the server.